src/ex1/script.js

```diff
@@ -0,0 +1,213 @@
+let tasks = [];
+let page_to_tasks_map = new Map();
+let last_page = 1;
+let current_page = 1;
+let chronological_arr = [];
+
+const addBtn = document.getElementById("push");
+const taskToAdd = document.getElementById("input");
+const pagination_element = document.getElementById("pagination");
+const lexiSortBtn = document.getElementById("lexiSort");
+const chronoSortBtn = document.getElementById("chronoSort");
+
+pagination_element.innerHTML = "";
+let btn = PaginationButton(current_page);
```

Comment:
which 'btn'? :) always think about the next developer who's going to work with you